README.md

```diff
@@ -1,2 +1,47 @@
-# Microverse-Project
-Hello microverse project.
+# Hello-Microverse
+
+> My First Official Microverse Repository 
+
+![screenshot](Screenshot%20(8).png)
```

Comment:
- [ ] Please remember All main headers from the [README template](https://github.com/microverseinc/readme-template) are expected to be present in your repo. I can see your readme misses some of the main headings including the `Future Features` heading, Please include 1-3 potential features to be added for the `Future Features`  along with other headers.  
- [ ] Also, the Live demo section is only required when you have a project with a live link, Hence you can remove it for this project.